generators/server/templates/src/main/java/package/config/_CacheConfiguration.java

```diff
@@ -40,13 +41,12 @@
 @SuppressWarnings("unused")
 @Configuration
 @EnableCaching
-@AutoConfigureAfter(value = { MetricsConfiguration.class<% if (databaseType == 'sql' || databaseType == 'mongodb') { %>, DatabaseConfiguration.class<% } %> })
+@AutoConfigureAfter(value = { MetricsConfiguration.class })
+@AutoConfigureBefore(value = { WebConfigurer.class<% if (databaseType == 'sql' || databaseType == 'mongodb') { %>, DatabaseConfiguration.class<% } %> })
 public class CacheConfiguration {
 
```

Comment:
I mean that instead of having a `private CacheManager cacheManager` you can have a local variable in the `cacheManager()` method.

Since I'm commenting code that isn't in the diff, it makes it a bit easier to be clear... Sorry about that.